File: backend/src/main/java/com/drip/admin/modules/system/vo/UserListVo.java
package com.drip.admin.modules.system.vo;

import java.time.LocalDateTime;
import java.util.List;

public record UserListVo(
    Long id,
    String username,
    String realName,
    String phone,
    String email,
    Integer status,
    DeptSummaryVo dept,
    List<RoleSummaryVo> roles,
    LocalDateTime createdAt,
    LocalDateTime lastLoginAt
) {
}

File: backend-go/internal/modules/system/vo/user_list_vo.go
package vo

import (
	"drip-admin/backend-go/internal/common"
	"time"
)

type UserListVo struct {
	ID          common.Int64String `json:"id"`
	Username    string             `json:"username"`
	RealName    string             `json:"realName"`
	Phone       *string            `json:"phone"`
	Email       *string            `json:"email"`
	Status      int                `json:"status"`
	Dept        *DeptSummaryVo     `json:"dept"`
	Roles       []RoleSummaryVo    `json:"roles"`
	CreatedAt   *time.Time         `json:"createdAt"`
	LastLoginAt *time.Time         `json:"lastLoginAt"`
}
